src/main/kotlin/net/leanix/githubagent/services/GitHubGraphQLService.kt

```diff
@@ -0,0 +1,77 @@
+package net.leanix.githubagent.services
+
+import com.expediagroup.graphql.client.spring.GraphQLWebClient
+import kotlinx.coroutines.runBlocking
+import net.leanix.githubagent.config.GitHubEnterpriseProperties
+import net.leanix.githubagent.dto.PagedRepositories
+import net.leanix.githubagent.dto.RepositoryDto
+import net.leanix.githubagent.exceptions.GraphQLApiException
+import net.leanix.githubagent.graphql.data.GetRepositories
+import net.leanix.githubagent.graphql.data.getrepositories.Blob
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+
+@Component
+class GitHubGraphQLService(
+    private val cachingService: CachingService,
+    private val gitHubEnterpriseProperties: GitHubEnterpriseProperties
+) {
+    companion object {
+        private val logger = LoggerFactory.getLogger(GitHubGraphQLService::class.java)
+        private const val PAGE_COUNT = 20
+        private const val MANIFEST_FILE_NAME = "leanix.yaml"
```

Comment:
not sure if we want to support other names, but at least leanix.yml should be valid